### PR TITLE
allow NavItem to be external links (removes console error within Link for shop)

### PR DESCRIPTION
**Pull request by saragibby**

### CODE REVIEW

File: components/Header/NavItem.js [hunk 1]
```diff
@@ -41,17 +43,35 @@ const NavItem = ({
...
+      ) : (
+        <StyledLink a={href} color={color} target={target}>
+          {displayedLink()}
+        </StyledLink>
+      )}
+    </>
   );
 };
 
+NavItem.propTypes = {
```
Comment (saragibby) [inline components/Header/NavItem.js]: Crap, forgot I started filling some of these in. I can complete the list.

Comment (saragibby) [reply components/Header/NavItem.js]: done

File: components/Header/NavItem.js [hunk 2]
```diff
@@ -41,17 +43,41 @@ const NavItem = ({
...
 
   return (
-    <Link href={href} passHref>
-      <StyledLink onClick={onClick} color={color} target={target}>
-        {displayedLink()}
-      </StyledLink>
-    </Link>
+    <>
+      {isLocal ? (
+        <Link href={href} passHref>
```
Comment (kburnell) [inline components/Header/NavItem.js]: @saragibby why does this need be a next link?

Comment (saragibby) [reply components/Header/NavItem.js]: Using `Link` keeps it from a full refresh and will just fill the content within `InnerPage`, so it's a nicer experience.

**Pull request merged**